content/en/blog/2022/exponential-histograms/index.md

```diff
@@ -0,0 +1,308 @@
+---
+title: "Exponential Histograms: Better Data, Zero Configuration"
+linkTitle: Exponential Histograms
+date: 2022-08-22
+author: "[Jack Berg](https://github.com/jack-berg)"
+canonical_url: https://newrelic.com/blog/best-practices/opentelemetry-histograms
+---
+
+Histograms are a powerful tool in the observability tool belt. OpenTelemetry
+supports histograms because of their ability to efficiently capture and transmit
+distributions of measurements, enabling statistical calculations like
+percentiles (for example, p90, p95, p99 representing the 90th, 95th, and 99th
+percentile).
```

Comment:
```suggestion
[percentiles](https://en.wikipedia.org/wiki/Percentile).
```

The parenthetical remark doesn't add much. Also, AFAICT, "p" followed by a number is standard notation for a percentile.